modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -105,3 +134,7 @@ $ gcloud compute target-pools add-instances \
 $ gcloud compute target-pools add-instances \
     ${INFRA_ID}-ign-target-pool --instances-zone="${ZONE_0}" --instances=${INFRA_ID}-bootstrap
 ----
+
```

Comment:
For Add the bootstrap instance to the load balancers step:
1. Add bootstrap instance to internal load balancer instance group:

    gcloud compute instance-groups unmanaged add-instances ${INFRA_ID}-bootstrap-instance-group --zone=${ZONE_0} --instances=${INFRA_ID}-bootstrap

2. Add bootstrap instance group to the internal load balancer backend service

    gcloud compute backend-services add-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-instance-group --instance-group-zone=${ZONE_0}